specification/storagepool/resource-manager/Microsoft.StoragePool/stable/2021-08-01/examples/IscsiTargets_Get.json

```diff
@@ -0,0 +1,59 @@
+{
+  "parameters": {
+    "api-version": "2021-08-01",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "diskPoolName": "myDiskPool",
+    "iscsiTargetName": "myIscsiTarget"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.StoragePool/diskPools/myDiskPool/iscsiTargets/myIscsiTarget",
+        "name": "myIscsiTarget",
+        "type": "Microsoft.StoragePool/diskPools/iscsiTargets",
+        "systemData": {
+          "createdAt": "2020-06-24T06:53:57+00:00",
+          "createdBy": "alias",
+          "createdByType": "User",
+          "lastModifiedAt": "2020-06-24T06:53:57+00:00",
+          "lastModifiedBy": "alias",
+          "lastModifiedByType": "User"
+        },
+        "properties": {
+          "targetIqn": "iqn.2005-03.org.iscsi:server1",
+          "aclMode": "Static",
+          "luns": [
+            {
+              "managedDiskAzureResourceId": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vm-name_DataDisk_1",
+              "name": "lun0",
+              "lun": 0
+            }
+          ],
+          "staticAcls": [
+            {
+              "initiatorIqn": "iqn.2005-03.org.iscsi:client",
+              "mappedLuns": [
+                "lun0"
+              ]
+            }
+          ],
+          "endpoints": [
+            "10.0.0.1:3260"
+          ],
+          "activeSessions": [
+            "iqn.2005-03.org.iscsi:client"
+          ],
+          "port": 3260,
+          "provisioningState": "Succeeded",
+          "status": "Healthy"
+        }
+      },
+      "managedBy": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.AVS/privateClouds/myPrivateCloud/clusters/Cluster-1/datastores/datastore1",
+      "managedByExtended": [
+        "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.AVS/privateClouds/myPrivateCloud/clusters/Cluster-1/datastores/datastore1",
+        "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.AVS/privateClouds/myPrivateCloud/clusters/Cluster-1/datastores/datastore2"
+      ]
```

Comment:
@jorgecotillo, 2 clarifying questions here:
1) Is it ok to add these properties outside of `diskpoolProperties `(`managedBy `is describe to be a top-level property in ARM RPC guidelines but not `managedByExtended`)
2) Looking at CRP's swagger, they mark these as read-only. Will we get flag by swagger correctness in the event when the owning rp (which will set these values) will be setting these (hence arm will see that they are not read-only). Is the read-only field here only used for the sdk/cli generation?

https://github.com/Azure/azure-rest-api-specs/blob/88e7838a09868a51de3894114355c75929847a46/specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json#L1344

Reply:
For #1 yes, it is ok, we have a ManagedByExtended which is an array of strings (I looked into the code to confirm), will update our RPC guidelines.

For #2 mark as readOnly: true the properties that your RP will set, if you have properties that a customer can set, then no need to mark them as readOnly: true to avoid getting flagged.

Reply:
For #1, thanks for making the update to RPC guideline.
For #2, to be clear, a different RP will be setting these values (not our RP). For all intent an purposes the other rp's request ar going through ARM so they are essentially "customer" request. Are you suggesting we make this field not read-only in that case? Now that I think about it, the disk example above might not be representative, I'm guessing CRP might be having some internal logic between vm / disk but the same RP is setting these value hence probably why they have that field as read-only.

Reply:
If you are allowing your customers (human, RP) to set a value during a PUT call, then do not mark the property as readOnly: true, if the property is set in the backend, then you can set readOnly: true. Hope is clear.

Reply:
@jorgecotillo can you also give guidance for who to contact since this LintDiff test is failing because it thinks managedByExtended is an unknown field?
![image](https://user-images.githubusercontent.com/24836340/130704494-d235b92a-c25c-4359-adef-e3c4849de68a.png)

Reply:
@changlong-liu can you help here?

Reply:
@jorgecotillo Great. That clarifies everything for me. Thank you.

Reply:
Hi @harshitha-akkaraju , would you please ping jianyxi or ruoxuan for the LintDiff concerns

Reply:
@jianyexi , @ruowan any tips on how to work around this lintdiff failure? `managedByExtended` is a valid top-level property.